### ISSUE TITLE
Embed scene diagram when exporting to clipboard

### ISSUE THREAD

**Issue by pomdtr** [excalidraw/excalidraw#3467]
The `Embed scene into exported file` option has no effect when exporting to clipboard. Is it a technical constraint ?
If not, I think it would be reasonable to let the user choose to embed the scene in the schema.

![image](https://user-images.githubusercontent.com/17577332/115166246-77d4f500-a0b2-11eb-8908-6644f8667430.png)

**Comment by leighmcculloch** [comment]
💯 Embedding the scene by default would be awesome, especially when using the Shift-Option-C shortcut to copy to clipboard. I always check the embed scene box, but there are odd times I forget to, or I save the image by using the screenshot to clipboard and then return later to edit it and can't because there's no scene data embedded.

**Comment by leandrotapia** [comment]
Hi there! I would like to vote for the scene to be embedded when copied to the clipboard.
My current sharing process is:
1. Download the image as a file (with embedded scene)
2. Upload it where I want to share it (slack, some Google document, etc.)
3. Check that the scene can be reconstructed from the image (because sometimes I copy it directly to the clipboard)
4. Lastly, delete the locally downloaded file
I think that including the scene in the clipboard could reduce this sharing process just by doing step 2, "just pasting it"

**Comment by arpitjp** [comment]
The embedded scene gets lost anytime the image gets compressed. I think the same is happening when copying to clipboard. Would be awesome if this limitation can be fixed.

**Comment by dwelle** [comment]
> The embedded scene gets lost anytime the image gets compressed. I think the same is happening when copying to clipboard. Would be awesome if this limitation can be fixed.

I think that's in the hands of the the compression tool, and stripping meta tags is likely what most compression tools do.

**Comment by bbil** [comment]
Yeah, this bit me today. If it's a technical constraint, then fair game. Just may need to make it clearer in the UI that "copy to clipboard" cannot use the embedded scene?

Workflow of saving as file, and then putting it where it needs to go (google doc / confluence) works just fine.

Just a bit of a foot gun that took me a bit to figure out what went wrong with my assumptions that "copy to clipboard" behaves the exact same way as the download buttons.

**Comment by pomdtr** [comment]
If you want a cleaner workflow, this specific issue was the root motivation behind building the vscode extension for excalidraw.

It allows you to edit *.excalidraw.png files directly from vscode, without continually import / export stuffs
